src/views/Main/index.jsx

```diff
@@ -73,7 +74,7 @@ class MainContainer extends Component {
 
     async getReportees(userSession, ldapEmail) {
       const secretsClient = userSession.getTaskClusterSecretsClient();
-      const partialOrg = await getAllReportees(secretsClient, ldapEmail);
+      const partialOrg = (userSession.oidcProvider === 'mozilla-auth0') ? await getAllFakeReportees(ldapEmail) : await getAllReportees(secretsClient, ldapEmail);
```

Comment:
Call `getAllReportees` with `userSession` and make this changes within `getAllReportees`:
https://github.com/mozilla/bugzilla-dashboard/blob/master/src/utils/getAllReportees.js#L52

If you do this you will not need the files `getAllFakeReportees` and `getOrgChart`.